files/image_config/hostcfgd/hostcfgd

```diff
@@ -166,10 +167,40 @@ class HostConfigDaemon:
             log_data['passkey'] = obfuscate(log_data['passkey'])
         syslog.syslog(syslog.LOG_INFO, 'value of {} changed to {}'.format(key, log_data))
 
+    def hostname_handler(self, key, data):
+        hostname = data["hostname"]
+        if hostname == self.hostname_cache:
+            return
+
+        syslog.syslog(syslog.LOG_INFO, "Get all running containers")
+        cmd = 'docker ps --format "{{.Names}}"'
+        try:
+            containers = subprocess.check_output(cmd, shell=True).split("\n")[:-1]
+        except subprocess.CalledProcessError as err:
+                syslog.syslog(syslog.LOG_WARNING, "{} - failed: return code - {}, output:\n{}"
+                                .format(err.cmd, err.returncode, err.output))
+
+        for name in containers:
+            script = '/usr/bin/{}.sh'.format(name)
+            exists = os.path.isfile(script)
+            if not exists:
+                syslog.syslog(syslog.LOG_INFO, "Can't find controll script for {}".format(name))
+                continue
+
+            cmd = "{} updateHostName {}".format(script, hostname)
```

Comment:
>hostname [](start = 56, length = 8)

Possible to inject attack from Redis, such as 'rm -rf /'

Reply:
Added validation